File: Bmi_2/app/src/main/java/com/pjatk/bmi_2/model/Answer.java
package com.pjatk.bmi_2.model;

public class Answer {
    private String _answerText;
    private boolean _isCorrect;

    public Answer(String answerText, boolean isCorrect){
        _answerText = answerText;
        _isCorrect = isCorrect;
    }

    public String getAnswerText(){
        return _answerText;
    }

    public  Boolean getIsCorrect(){
        return _isCorrect;
    }
}

File: BMICalculator/app/src/main/java/com/szymon/bmicalculator/model/Answer.kt
package com.szymon.bmicalculator.model

class Answer(answerText: String, isCorrect: Boolean) {
    private var _answerText: String = ""
    private var _isCorrect: Boolean = false

   init {
       _answerText = answerText
       _isCorrect = isCorrect
   }
    fun getAnswerText(): String {
        return _answerText
    }

    fun getIsCorrect(): Boolean? {
        return _isCorrect
    }
}